web_src/less/_repository.less

```diff
@@ -940,44 +940,30 @@
                     }
                 }
 
+                &::before {
+                    content: "";
+                    float: left;
+                    margin-left: -36px;
+                    border-radius: 15px;
+                    background-color: whitesmoke;
+                }
+
                 .svg {
                     width: 30px;
+                    height: 30px;
                     float: left;
                     text-align: center;
+                    line-height: 30px;
+                    margin-left: -36px;
+                    padding: 7px;
 
                     &.octicon-circle-slash {
-                        margin-top: 5px;
-                        margin-left: -35.5px;
-                        height: 20px;
                         color: #bd2c00;
                     }
 
                     &.octicon-primitive-dot {
-                        margin-top: -1px;
-                        margin-left: -35.5px;
-                        margin-right: -1px;
-                        height: 30px;
                         color: #6cc644;
                     }
-
-                    &.octicon-bookmark {
-                        margin-top: 2px;
-                        margin-left: -35.5px;
```

Comment:
Hello, I don't thinks It's necessary to delet them because we should change the size base on the type . only delet `` margin-left: -35.5px;`` is  enough. Do you think so, thanks

Reply:
margin-top: 2px; and height: 25px; will move icon out of background center, margin-left is redundant, margin-right: -1px; seems to do nothing so i removed it all together